1.1.c

```diff
@@ -0,0 +1,100 @@
+#include <string.h>
+#include <stdio.h>
+
+#define binarySize 256
+
+
+int transformation2ndWay(char* binary, char* octale)	//https://calcus.ru/perevod-sistem-schisleniya/iz-dvoichnoy-v-vosmerichnuyu here i took a way to transformation (2nd way)

```

Comment:
```suggestion
int blockOfNumberTransformation(char* binary, char* octale)	//https://calcus.ru/perevod-sistem-schisleniya/iz-dvoichnoy-v-vosmerichnuyu here i took a way to transformation (2nd way)
```
Название должно соответствовать смыслу функции (преобразование блока числа), а не источнику, где алгоритм описан
(-0.5)